Documentation/subcommands/run.md

```diff
@@ -309,32 +309,32 @@ If you don't use systemd, you can use [daemon](http://www.libslack.org/daemon/)
-| `--stage1-path` |  `` | A path to a stage1 image. Absolute and relative paths are supported | Image to use as stage1 |
-| `--stage1-name` |  `` | A name of a stage1 image. Will perform a discovery if the image is not in the store | Image to use as stage1 |
-| `--stage1-hash` |  `` | A hash of a stage1 image. The image must exist in the store | Image to use as stage1 |
-| `--stage1-from-dir` |  `` | A stage1 image file inside the default stage1 images directory | Image to use as stage1 |
-| `--store-only` |  `false` | `true` or `false` | Use only available images in the store (do not discover or download from remote URLs). See [image fetching behavior](../image-fetching-behavior.md) |
-| `--uuid-file-save` |  `` | A file path | Write out the pod UUID to a file |
-| `--volume` |  `` | Volume syntax (`NAME,kind=KIND,source=PATH,readOnly=BOOL`). See [Mount Volumes into a Pod](#mount-volumes-into-a-pod) | Volumes to make available in the pod |
+| `--cpu` | none | CPU units (ex. `--cpu=500m`) | CPU limit for the preceding image in [Kubernetes resource model](http://kubernetes.io/v1.1/docs/design/resources.html) format. |
+| `--dns` | none | IP Address | Name server to write in `/etc/resolv.conf`. It can be specified several times |
+| `--dns-opt` |  none | DNS option  | DNS option from in resolv.conf(5) to write in `/etc/resolv.conf`. It can be specified several times. |
+| `--dns-search` |  none | Domain name | DNS search domain to write in `/etc/resolv.conf`. It can be specified several times. |
+| `--exec` | none | Path to executable | Override the exec command for the preceding image. |
+| `--inherit-env` |  `false` | `true` or `false` | Inherit all environment variables not set by apps. |
+| `--interactive` |  `false` | `true` or `false` | Run pod interactively. If true, only one image may be supplied. |
+| `--mds-register` |  `false` | `true` or `false` | Register pod with metadata service. It needs network connectivity to the host (`--net` as `default`, `default-restricted`, or `host`). |
+| `--memory` |  none | Memory units (ex. `--memory=50M`) | Memory limit for the preceding image in [Kubernetes resource model](http://kubernetes.io/v1.1/docs/design/resources.html) format. |
+| `--mount` |  none | Mount syntax (ex. `volume=NAME,target=PATH`) | Mount point binding a volume to a path within an app. See [Mounting Volumes without Mount Points](#mounting-volumes-without-mount-points). |
+| `--net` |  `default` | A comma-separated list of networks. (ex. `--net[=n[:args], ...]`) | Configure the pod's networking. Optionally, pass a list of user-configured networks to load and set arguments to pass to each network, respectively. |
+| `--no-overlay` |  `false` | `true` or `false` | Disable the overlay filesystem. |
+| `--no-store` |  `false` | `true` or `false` | Fetch images, ignoring the local store. See [image fetching behavior](../image-fetching-behavior.html) |
```

Comment:
Changing .md to .html breaks the link when the page is read directly from https://github.com/coreos/rkt/blob/master/Documentation/subcommands/run.md